tests/llmcompressor/transformers/sparsification/test_compress_tensor_utils.py

```diff
@@ -159,9 +162,14 @@ def test_quant_model_reload(format, dtype, tmp_path):
     output_dir = tmp_path / "oneshot_out"
     splits = {"calibration": "train[:10%]"}
 
+    model = AutoModelForCausalLM.from_pretrained(
+        model_path,
+        torch_dtype=dtype,
+    )
+
     # create a quantized model
     oneshot(
-        model=model_path,
+        model=model,
```

Comment:
you should still be able to pass in a string

Reply:
You can but the main pathway to run compression and save is to use model.save_pretrained() outside the oneshot. Hence passing in model